migrations/2025-02-06-233105_remove_post_sort_type_enums/up.sql

```diff
@@ -0,0 +1,69 @@
+-- This removes all the extra post_sort_type_enums,
+-- and adds a default_post_time_range_seconds field.
+-- Change all the top variants to top in the two tables that use the enum
+-- Because of a postgres bug, you can't assign this to a new enum value,
+-- unless you run an unsafe commit first. So just use active.
+-- https://dba.stackexchange.com/questions/280371/postgres-unsafe-use-of-new-value-of-enum-type
+UPDATE
+    local_user
+SET
+    default_post_sort_type = 'Active'
+WHERE
+    default_post_sort_type IN ('TopDay', 'TopWeek', 'TopMonth', 'TopYear', 'TopAll', 'TopHour', 'TopSixHour', 'TopTwelveHour', 'TopThreeMonths', 'TopSixMonths', 'TopNineMonths');
+
+UPDATE
+    local_site
+SET
+    default_post_sort_type = 'Active'
```

Comment:
This and the one above should be `Top`, not `Active`

Reply:
Yep. I commented on this, but I ran into a weird postgres bug, where if you add a new enum value, you can't use it unless you commit the transaction. That seems too dangerous to do with sql migrations (which are already transactions), so its safer to just use an existing value, Active. 

Its not too big a deal, since people with `TopX` sorts will have to manually change them anyway.

Reply:
Yes that is true, being switched from Top Day to Top (all time) doesnt make much sense anyway.